src/ToDo.UI/Views/HomePage.xaml

```diff
@@ -98,37 +98,37 @@
 				</Grid>
 			</muxc:NavigationView.PaneHeader>
 
-		<!-- Task Lists: template for WellKnownLists & CustomLists -->
-		<muxc:NavigationView.MenuItemTemplate>
-			<DataTemplate>
-				<muxc:NavigationViewItem Content="{Binding DisplayName}"
+			<!-- Task Lists: template for WellKnownLists & CustomLists -->
+			<muxc:NavigationView.MenuItemTemplate>
+				<DataTemplate>
+					<muxc:NavigationViewItem Content="{Binding DisplayName}"
 										 uen:Navigation.Data="{Binding}">
-					<muxc:NavigationViewItem.Icon>
-						<PathIcon Data="{Binding Converter={StaticResource TaskListIconValueConverter}}"
+						<muxc:NavigationViewItem.Icon>
+							<PathIcon Data="{Binding Converter={StaticResource TaskListIconValueConverter}}"
 								  Foreground="{Binding Converter={StaticResource TaskListIconForegroundConverter}}" />
```

Comment:
if you are re-aligning the xaml (which is fine), make sure the attributes are re-aligned equally as well
& same for line 108+118